### DISCUSSION TITLE
Übergabe Ziel- Zwischenziel in WunderlinQ App / Transfer destination - interim destination in WunderlinQ App

### DISCUSSION THREAD

**Discussion by ToToGSARider** [devemux86/cruiser#226] [Q&A] [answered]
Es gibt im BMW Navigator Felder die man einblenden kann, wo man die Kilometer / Zeit bis Ziel / Zwischenziel sieht. Ist es technisch möglich diese Daten in die WunderlinQ App zu übergeben? Es müsste natürlich auch eine Anpassung von blackboxembedded an der WunderlinQ App vorgenommen werden. Ich fände es ein cooles Feature, da ja in der Cruiser App nur Ziel oder Zwischenziel eingeblendet werden kann. 

There are fields in the BMW Navigator that can be displayed, where you can see the kilometers / time to destination / stopover. Is it technically possible to transfer this data to the WunderlinQ app? Of course there would have to be a customization of blackboxembedded to the WunderlinQ app. I think it would be a cool feature, because in the Cruiser app you can only see the destination or the waypoint. 

 

![Unbenannt](https://github.com/devemux86/cruiser/assets/128730865/fdcead43-1b52-4b0b-9c8e-aa3b01fc6e7a)

**Comment by devemux86** [comment, marked as answer]
@ToToGSARider thanks for the idea!

Do you mean while driving, the distance + time to next waypoint and destination,
to be passed to the motorcycle via the WunderLINQ?

We need to hear from @blackboxembedded if / how this is possible via the WunderLINQ,
then I can see what can be done from the Cruiser side.

---

@ToToGSARider Danke für die Idee!

Meinen Sie während der Fahrt, die Entfernung + Zeit bis zum nächsten Wegpunkt und Ziel,
über die WunderLINQ an das Motorrad weitergegeben werden?

Wir müssen von @blackboxembedded hören, ob / wie dies über die WunderLINQ möglich ist,
dann kann ich sehen, was auf der Cruiser-Seite möglich ist.

**Reply by ToToGSARider** [reply]
@devemux86 Die Daten müssten aus der Cruiser App in die WunderlinQ App übergeben werden. Das Motorrad ist hier komplett außen vor. Es wäre ein reiner Austausch / Abgleich der Apps untereinander. Wenn man zum Beispiel im Navigator eine Route gefahren ist, konnte man zu diesem Menü gehen und hat dort diese Infos gesehen. So könnte man in der Cruiser App oben rechts den Platz einsparen und dort zum Beispiel wie beim Navigator nur die Höhe und Akkustand einblenden. Man kann ja in der App von WunderlinQ einstellen, welche Datenfelder zu sehen sein sollen. Die Datenfelder für Zeiten und Ziele sind dort noch nicht vorgesehen, weil bestimmt noch keiner danach gefragt hat.  Wie gesagt, nur eine Idee :-) 

@devemux86 The data would have to be transferred from the Cruiser app to the WunderlinQ app. The motorcycle is completely out of the picture here. It would be a pure exchange / alignment of the apps with each other. For example, if you drove a route in Navigator, you could go to that menu and see that info there. So you could save the space in the top right corner of the Cruiser app and only show the altitude and battery status, for example, as in the Navigator. You can set in the app of WunderlinQ, which data fields should be seen. The data fields for times and destinations are not yet provided there, because certainly no one has asked for it yet.  As I said, just an idea :-)
